README.rst

```diff
@@ -80,7 +80,7 @@ corresponding image pull tags and their release notes where applicable:
 +--------------------------------------------------------------------------------------+------------+----------------------------------------------+------------------------------------------------------------------------------------------+
 | `main <https://github.com/cilium/cilium/commits/main>`__                             | daily      | ``quay.io/cilium/cilium-ci:latest``          | N/A                                                                                      |
 +--------------------------------------------------------------------------------------+------------+----------------------------------------------+------------------------------------------------------------------------------------------+
-| `v1.14.0-rc.1 <https://github.com/cilium/cilium/commits/v1.14.0-rc.1>`__             | 2023-07-17 | ``quay.io/cilium/cilium:v1.14.0-rc.1``       | `Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.14.0-rc.1>`__ |
+| `v1.14.1-rc.1 <https://github.com/cilium/cilium/commits/v1.14.1-rc.1>`__             | 2023-07-17 | ``quay.io/cilium/cilium:v1.14.1-rc.1``       | `Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.14.1-rc.1>`__ |
```

Comment:
I guess this got caught up in some `sed` expression, but there's no such release:

```suggestion
| `v1.14.0-rc.1 <https://github.com/cilium/cilium/commits/v1.14.1-rc.1>`__             | 2023-07-17 | ``quay.io/cilium/cilium:v1.14.1-rc.1``       | `Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.14.0-rc.1>`__ |
```

Reply:
Now that we have 1.14.1, should we remove this section?

Reply:
This particular line will be updated in about a week when we do the first v1.15.0-pre.X release.

Reply:
So feel free to either drop it or just ignore this line for now.

Reply:
Left it as is